<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>488</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="paperWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>681</width>
       <height>441</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>论文题目：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>论文作者：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>发表期刊：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>54</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>金额：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>330</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>影响因子(IF)：</string>
      </property>
     </widget>
     <widget class="QLabel" name="paperTitleEdit_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="paperMoneyAmountEdit_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>250</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="paperAuthorEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="paperIFEdit_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>330</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="paperJournalsEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="prizeWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>681</width>
       <height>441</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_41">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>奖项名称：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_42">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>赞助商：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_43">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>获奖人：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_44">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>350</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>金额：</string>
      </property>
     </widget>
     <widget class="QLabel" name="prizeNameEdit_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>45</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="prizeAwardingCompanyEdit_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>145</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="prizeWinnerEdit_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>250</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="prizeMoneyAmountEdit_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>350</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="projectWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>681</width>
       <height>441</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_45">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>项目名称：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_46">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>委托单位：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_47">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>负责人：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_48">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>300</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>项目经费：</string>
      </property>
     </widget>
     <widget class="QLabel" name="projectNameEdit_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>65</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="projectMoneyAmountEdit_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>300</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="projectPrincipalEdit_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="projectRequesterEdit_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="monographWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>681</width>
       <height>441</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_49">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>专著名称：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_50">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>专著作者：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_51">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>320</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>专著金额：</string>
      </property>
     </widget>
     <widget class="QLabel" name="monographNameEdit_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="monographMoneyAmountEdit_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>310</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="monographAuthorEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="preferPageButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>443</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>上一页</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>相关图片：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>业绩工分：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>410</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>备注：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>业绩发生时间：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>业绩类型：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>教师ID：</string>
     </property>
    </widget>
    <widget class="QLabel" name="relatedPicEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>280</y>
       <width>271</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="perforTypeEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="teacherIDEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>150</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="creditEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="noteEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>410</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="happenTimeEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="nextPageButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>450</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>下一页</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
